ivy/container/experimental/layers.py

```diff
@@ -942,3 +942,67 @@ def dct(
             norm=norm,
             out=out,
         )
+
```

Comment:
Could you also write the docstring example for both container instance and static methods